sources/packages/web/src/helpers/questions-financial.ts

```diff
@@ -0,0 +1,16 @@
+export const questionsFin = {
```

Comment:
I do not see a problem moving the questions, but I would prefer that they are moved inside a specific folder for fa-application, and please move all others also.

Reply:
@andrewsignori-aot @2011sandeepraj 
1. I've created one [technical debt](https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/188) to manage static texts of the application and questionnaire.
2.  We have one placeholder for design challenge on questionnaire model, that may change whole section.  So, I will recommend, keep this portion in view/component.